Logger/IEntity.cs

```diff
@@ -1,7 +1,7 @@
-﻿namespace Logger;
+namespace Logger;
 public interface IEntity
 {
-
         // Place members here.
-
+        public Guid ID { get;  init; }
+        public abstract string Name { get; }
```

Comment:
Interfaces should not have accessibility modifiers
```suggestion
        Guid ID { get;  init; }
        abstract string Name { get; }
```